tests/test_hcp.py

```diff
@@ -22,14 +22,14 @@
 class TestProcess(unittest.TestCase):
 
     def test01_upload_file(self):
-        self.assertIsNone(hcpm.upload_file(f"{testWD}/data/test_reads_R1.fasterq", "unittest/test_reads_R1.fasterq"))
+        self.assertIsNone(hcpm.upload_file(f"{testWD}/data/test_reads_R1.fasterq", "unittest/test_reads_R1.fasterq",silent=True))
```

Comment:
Will these petagene compressed test files work on a system that doesn't have petagene installed?

Reply:
It transfers something. But idk. Can just replace with fastq if desired